examples/notebooks/6_Noise_Simulation.ipynb

```diff
@@ -606,16 +716,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 24,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "QuantumError([(<qiskit.circuit.quantumcircuit.QuantumCircuit object at 0x7f6f97b34a90>, 1.0)])"
+       "QuantumError([(<qiskit.circuit.quantumcircuit.QuantumCircuit object at 0x7f20042ae880>, 1.0)])"
```

Comment:
We should avoid printing directly the error, because the __repr__ of Qiskit is not clean, and shows the address in memory of the object. I would print something else that is still interesting